src/Displaz.jl

```diff
@@ -6,6 +6,8 @@ using Colors
 export plot3d, plot3d!, clearplot, viewplot
 export KeyEvent, CursorPosition, event_loop
 
+displaz_cmd() = get(ENV, "DISPLAZ_CMD", "displaz")
```

Comment:
Yeah, not a big deal.  The right thing for (2) is simply to set it in `__init__()`, correct?